src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -291,6 +293,32 @@ impl HummockEventHandler {
         // Notify completion of the Clear event.
         notifier.send(()).unwrap();
     }
+
+    fn handle_version_update(&self, version_payload: Payload) {
+        let max_committed_epoch_before_update =
+            self.local_version_manager.get_pinned_version().id();
```

Comment:
Read lock is acquired here. Since the return is only used after `try_update_pinned_version`, which acuires a lock already, can we avoid getting the MCE here to avoid one lock acqusition?

Reply:
good idea,  i fix it through return mce_change flag to avoid one lock acqusition